src/lesson3/LessonThree.java

```diff
@@ -0,0 +1,155 @@
+            System.out.print(b + " ");
+        }
+    }
+
+    public static void getPopulationSize() {
+        int currentPopulation = 10_000_000;
+        float birthRate = 0.014f, deathRate = 0.008f;
+        for (int i = 1; i <= 10; i++) {
+            currentPopulation += (currentPopulation * birthRate) - (currentPopulation * deathRate);
+        }
+        System.out.println(currentPopulation);
+    }
+
+    public static void getFinalDeposit(Scanner scanner) {
+        System.out.print("Enter a deposit amount: ");
+        double depositSum = scanner.nextDouble();
+        System.out.print("Enter a number of months: ");
+        int months = scanner.nextInt();
+        float percent = 0.07f;
+        for (int i = 1; i <= months; i++) {
+            depositSum += depositSum * percent;
+        }
+
+//        int i = 1;
+//        while (i <= months){
+//            depositSum += depositSum * percent;
+//            i++;
+//        }
+        System.out.println("The final amount of the deposit for " + months + " months: " + depositSum);
+    }
+
+    public static void getDayOfWeek(Scanner scanner) {
+        int attempt = 0;
+        while (attempt != 3) {
+            System.out.print("Enter a number from 1 to 7: ");
+            int number = scanner.nextInt();
+            if (number < 1 || number > 7) {
+                System.out.println("Enter a number from 1 to 7.");
+            } else {
+                if (number == 1) {
+                    System.out.println("Monday");
+                } else if (number == 2) {
+                    System.out.println("Tuesday");
+                } else if (number == 3) {
+                    System.out.println("Wednesday");
+                } else if (number == 4) {
+                    System.out.println("Thursday");
+                } else if (number == 5) {
+                    System.out.println("Friday");
+                } else if (number == 6) {
+                    System.out.println("Saturday");
+                } else {
+                    System.out.println("Sunday");
+                }
+
+//                switch (number) {
+//                    case 1:
+//                        System.out.println("Monday");
+//                        break;
+//                    case 2:
+//                        System.out.println("Tuesday");
+//                        break;
+//                    case 3:
+//                        System.out.println("Wednesday");
+//                        break;
+//                    case 4:
+//                        System.out.println("Thursday");
+//                        break;
+//                    case 5:
+//                        System.out.println("Friday");
+//                        break;
+//                    case 6:
+//                        System.out.println("Saturday");
+//                        break;
+//                    case 7:
+//                        System.out.println("Sunday");
+//                        break;
```

Comment:
За умовою трохи інакше.
```suggestion
//                    case 6:
//                    case 7:
//                        System.out.println("Holiday");
//                        break;
```